lottie-ios/Classes/Extensions/UIColor+Expanded.m

```diff
@@ -293,6 +305,12 @@ - (NSString *)LOT_stringFromColor {
 		case kCGColorSpaceModelMonochrome:
 			result = [NSString stringWithFormat:@"{%0.3f, %0.3f}", self.white, self.alpha];
 			break;
+        case kCGColorSpaceModelLab:
+        case kCGColorSpaceModelCMYK:
+        case kCGColorSpaceModelDeviceN:
+        case kCGColorSpaceModelIndexed:
+        case kCGColorSpaceModelPattern:
+        case kCGColorSpaceModelUnknown:
 		default:
```

Comment:
Just curious - what is the advantage of including all the enum cases, but leaving the `default` case? I would think that if you wanted to be explicit about the cases, then you should remove the default case. Or leave the default case, and don't both enumerating the unneeded cases.